app/uk/gov/hmrc/pillar2/models/ApiErrors.scala

```diff
@@ -19,3 +19,4 @@ package uk.gov.hmrc.pillar2.models
 trait ApiErrors extends Throwable
```

Comment:
Have you considered making this sealed so future devs put all the different types of ApiErrors in one place?